src/js/components/droplist/index.js

```diff
@@ -133,8 +139,8 @@ export default Component.extend({
     this.popRegion.empty();
   },
   onPicklistSelect({ model }) {
-    this.setState('selected', model);
     this.popRegion.empty();
+    this.setState('selected', model);
```

Comment:
When a selection on one droplist spawned another emptying after the event would close the 2nd droplist